neptune/new/metadata_containers/metadata_container.py

```diff
@@ -338,7 +343,14 @@ def _define_attribute(self, _path: List[str], _type: AttributeType):
     def _get_root_handler(self):
         return Handler(self, "")
 
+    def get_url(self) -> str:
+        """Returns the URL that can be accessed within the browser"""
+        return self._url
```

Comment:
I don't really see the advantage of this vs simply having abstract `get_url` and implementing it in subclasses. What am I missing?

Reply:
discussed: the advantage is defining docstring once :)